### PR TITLE
Add Calculator Components

### PR DESCRIPTION
**Pull request by CollinsTatang**
In this branch, I have created the calculator UI
- Use class-based components.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
+  border: none;
+  height: 48px;
+  background-color: white;
+}
+
+.span-2-col {
+  grid-column: 1 / span 2;
+}
+
+.bg-orange {
```
Comment (Dipeshtwis) [inline src/index.css]: - [x] The design is not looking the same. There is one more div between the input field and all the buttons. I would recommend making a similar design.


### Your design

Screenshot from 2021-08-11 00-36-04


### Given design

Screenshot from 2021-08-10 20-48-42

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,5 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Calculator />;
```
Comment (Dipeshtwis) [inline src/App.js]: - [x] The `App.js` component should be class-based as per the project's requirements even though functional component is very correct. Please change this functional component to a class component. :+1:
- [x] You also need to import `React, {Component}` when you convert the functional component to class component extended from `React Component`.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @CollinsTatang, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Linters are passing :white_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by CollinsTatang** [comment]
Hi @Dipeshtwis and @arslanbisharat 
Thanks for the review.

**Pull request merged**